deploy/examples/csi/cephfs/storageclass-ec.yaml

```diff
@@ -14,10 +14,9 @@ parameters:
 
   # Ceph pool into which the volume shall be created
   # Required for provisionVolume: "true"
-
   # For erasure coded pools, we have to create a replicated pool as the default data pool and an erasure-coded
   # pool as a secondary pool.
-  pool: myfs-ec-data1
+  pool: myfs-ec-erasurecoded
```

Comment:
There are also some examples in `nfs.yaml` and `nfs-test.yaml` IIRC. Those might have been gone for a bit, so rebasing and doing a search for `myfs-data0` again might help

Reply:
`nsf.yaml` contains next:
```
  rados:
    # RADOS pool where NFS configs are stored.
    # In this example the data pool for the "myfs" filesystem is used.
    # If using the object store example, the data pool would be "my-store.rgw.buckets.data".
    # Note that this has nothing to do with where exported file systems or object stores live.
    pool: myfs-data0
```

So i guess, it is clear, why used such name, isn't it?